addon/components/sl-menu.js

```diff
@@ -572,41 +518,39 @@ export default Ember.Component.extend({
         }
 
         return extraClassNamesString;
-    }.property( 'extraClassNames' ),
+    }),
 
     /**
      * Boolean representation of showAll property
      *
-     * @function showAllBoolean
-     * @observes showAll
-     * @returns  {boolean}
+     * @function
+     * @returns {Boolean}
      */
-    showAllBoolean: function() {
+    showAllBoolean: Ember.computed( 'showAll', function() {
         return ( 'true' === this.get( 'showAll' ) ) ? true : false;
```

Comment:
Agreed. Are we making refactors in this changeset unrelated to CLI upgrade and the sl-grid though?